README.md

```diff
@@ -68,19 +68,13 @@ To import an individual component (for example, a button), add the line:
 ```
 
 
-To resolve your `@import` declarations you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) or [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths) in Ruby.
+To resolve your `@import` declarations you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) or [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths) in Ruby if you don't have it set up already.
 
-Below is a code sample you can add to your gulp configuration file using the gulp-sass package:
-```JS
-.pipe(sass({ includePaths: 'node_modules/' }))
-```
+If the component depends on image assets, there are 2 ways to reference them.
 
-To import images, configure your application to reference or copy the icons assets.
+For prototyping, you can reference the `icons` package in `node_modules` directly. The path for that would be `/node_modules/@govuk-frontend/icons`.
```

Comment:
I don't understand what we're trying to tell the user here. Where should they reference it? How? If we're talking about the _prototype kit_, shouldn't this be set up for them already?

Reply:
Not meaning protoype kit. More in a sense of `if you just want to set it up for dev this is the path, your app/build configuration aside`